knowledge/guidelines.rst

```diff
@@ -43,6 +43,9 @@ Good practices
   the graph to avoid the conflicts without overrides and forces is the recommended approach.
 - **Please, do not abuse 'tool_requires'**. Those are intended only for executables like ``cmake`` and ``ninja`` running in the "build"
   context, not for libraries or library-like dependencies, that must use ``requires`` or ``test_requires``.
+- Named arguments which receive a value should use ``=`` without spaces. For example, ``-r=conancenter`` is correct, but ``-r conancenter`` is not.
+- Positional arguments when invoking Conan should be specified first, before any named argument. For example,
+  ``conan install . -s="os=Windows"`` is correct, but ``conan install -s="os=Windows" .`` is not.
```

Comment:
Not fully sure. The ``-r conancenter`` does work, so maybe we want to reword a bit? I'd do:
- Use just 1 item to define both good practices for CLI args, not 2 bullet items
- Define the good practice without saying that the other is not correct (positive recommendation), just recommend one over the other (it can be said that there can be arg parsing errors if not in some cases)